recipes/morphometrics-engine/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "morphometrics-engine" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/morphometrics-engine-{{ version }}.tar.gz
+  sha256: f845c1bcb2334ade757ce19d88508027a00142e9c25a8a59d0d4585c06586c26
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - napari
+    - napari-skimage-regionprops
+    - numpy
+    - magicgui
+    - pandas
+    - qtpy
+    - superqt
+    - tqdm
+    - toolz
+
+test:
+  imports:
+    - morphometrics_engine
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/morphometrics/morphometrics-engine
+  summary: A morphometrics measurement engine.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - goanpeca
```

Thanks, @goanpeca ! Would it be okay for @haesleinhuepf and I to be added as maintainers? 

```suggestion
    - kevinyamauchi
    - haesleinhuepf
    - goanpeca
```

---

👍